test/test_common.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import os
+from pathlib import Path
+import sys
+
+BASE_DIR = Path(__file__).parent.parent
+PKG_DIR = BASE_DIR / 'packages'
+
+# TODO: remove once we have a proper Python package for common functions
+sys.path.append(str(BASE_DIR / 'tools'))
+import common  # noqa
+
+
+def registered_packages():
+    packages = [name for name in os.listdir(PKG_DIR)
+                if (PKG_DIR / name).is_dir()]
+    return packages
```

Comment:
Maybe add another helper function that yields the parsed `meta.yaml` files?  I can see other tests wanting to do that.

Reply:
Ah, I hadn't thought of the naming issue.  Probably fine without the extra helper function then.